tests/tests/swfs/avm2/movieclip_goto_scene_last_frame_label/Test.as

```diff
@@ -0,0 +1,22 @@
+package {
+import flash.display.MovieClip;
+import flash.events.Event;
+
+public class Test extends MovieClip {
+    private var lastFrame:Array = [];
+
+    public function Test() {
+        var that:Test = this;
+        that.currentScene.name
+        stage.addEventListener(Event.ENTER_FRAME, function (event:Event):void {
+            var currentFrame = [that.currentScene.name, that.currentFrame];
+            if (that.lastFrame != currentFrame) {
```

Comment:
Hmm, doesn't this compare by pointer, not contents?

Reply:
Removed it, as it was unnecessary anyways